gcalbot/gcalbot/handler.go

```diff
@@ -62,10 +59,15 @@ func (h *Handler) HandleCommand(msg chat1.MsgSummary) error {
 		return nil
 	}
 
-	tokens, err := shellquote.Split(cmd)
+	tokens, userErr, err := base.SplitTokens(cmd)
```

Comment:
@joshblum let me know what you think of this. I can also push down the kbc, convid and logger and have the helper handle that too